cfgov/unprocessed/apps/youth-employment-success/css/main.less

```diff
@@ -113,6 +113,11 @@ textarea {
   border-style: solid !important;
   border-width: 1px !important;
 }
+
+input:-moz-ui-invalid {
+  // Required to stop FF from validating the `placeholder` attribute
```

Comment:
What a strange hack!

Reply:
Yeah this was definitely a weird one, I suppose for a browser bug this isnt really a deal-breaker 😄